### PR TITLE
DB Seeds

### PR DESCRIPTION
**Pull request by beeequeue**
- Adds data to use when testing.
- Adds `thumbnail` to `Boardgame`

waiting for #28 and #33

```
------------------------------------------
Inserted 5 board games!
Scythe, Azul, Terraforming Mars (mock), Wingspan (mock), Gloomhaven (mock)
------------------------------------------
Inserted 25 users!
------------------------------------------
Created 5 groups of friends!
------------------------------------------
Created 170 matches!
------------------------------------------
```

### CODE REVIEW

File: packages/server/src/modules/match/match.resolvers.ts [hunk 1]
```diff
@@ -31,7 +31,7 @@ export class MatchResolver {
       throw new Error('Not found!')
     }
 
-    const improvedSchema = game.resultSchema
+    const improvedSchema = game.resultSchema as JsonSchemaObject
```
Comment (beeequeue) [inline packages/server/src/modules/match/match.resolvers.ts]: This type is incorrect, it doesn't have `$schema` at its base

Comment (beeequeue) [reply packages/server/src/modules/match/match.resolvers.ts]: The type of the class property should be MinimumSchema or something like that probably

Comment (peoplenarthax) [reply packages/server/src/modules/match/match.resolvers.ts]: As we talked, if we just use the metadata/playerResults property we won't need the `$schema` here anylonger.

File: packages/server/src/modules/boardgame/boardgame.model.ts [hunk 2]
```diff
@@ -24,10 +24,11 @@ type BoardgameConstructor = OptionalUuid<
   Pick<
     Boardgame,
     | 'uuid'
+    | 'type'
     | 'name'
     | 'shortName'
     | 'aliases'
-    | 'type'
```
Comment (beeequeue) [inline packages/server/src/modules/boardgame/boardgame.model.ts]: ?

**Pull request merged**